src/modules/m_sslinfo.cpp

```diff
@@ -373,8 +373,24 @@ class ModuleSSLInfo
 		{
 			OperInfo* ifo = i->second;
```

Comment:
This can be moved down to after the MatchFP check.

Reply:
The MatchFP check depends on the fingerprint which is read from the oper block's OperInfo, no?

Reply:
Ah, I'm an idiot. Yes, never mind.